### PR TITLE
CNFT11-402 Modernization Readme Update

### PR DESCRIPTION
**Pull request by Emelias-Enquizit**

### CODE REVIEW

File: libs/database-entities/README.md [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+
+The database-entities library is currently being used by both  the mondernization-api and patient-listener projects Some packages to note include:
+
+- gov.cdc.nbs.address - Location records for Country, County and City.
+- gov.cdc.nbs.audit -   Contains classes related to auditing a record.
+- gov.cdc.nbs.entity - Contains entity classes to interact with MS SQL DBMS and ElasticSearch. Also
+contains entity related enums and converters. 
+- gov.cdc.nbs.patient - Contains interface with commands related to patient record actions.
+- gov.cdc.nbs.repository - Contains repository interfaces for MS SQL Dialect and ElasticSearch.
+- gov.cdc.nbs.util - Contains helpful util classes and cosntants.
```
Comment (mpeels) [inline libs/database-entities/README.md:16]: `cosntants` typo

File: libs/event-schema/README.md [hunk 2]
```diff
@@ -0,0 +1,28 @@
+# Event-Schema
+
+Library that contains message based event classes usseful for Kafka event based comminucation between the
+modernization-api and patient-listener projects. Also contains useful classes for comminucation between modernization-api and mondernization-ui
```
Comment (mpeels) [inline libs/event-schema/README.md:4]: Couple typos:
`comminucation`, `mondernization`

File: libs/database-entities/README.md (hunk 1 above)
Comment (Emelias-Enquizit) [reply libs/database-entities/README.md]: updated.

File: libs/event-schema/README.md (hunk 2 above)
Comment (Emelias-Enquizit) [reply libs/event-schema/README.md]: updated.

**Pull request merged**